app/views/compute_resources/form/_vmware.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= javascript "application" %>
```

Comment:
i wonder why? as the app.js should already be loaded? maybe missing on content load event?